storage-resize-images/extension.yaml

```diff
@@ -320,6 +320,20 @@ params:
       - label: No
         value: false
 
+  - param: REGENERATE_TOKEN
+    label: Assign new access token
+    description: >
+      Should resized images have a new access token assigned to them,  different
+      from the original image?
+    type: select
+    required: true
```

Comment:
i don't think this should be required, it might cause a breaking change for existing users, change it to none-required.